datafusion/physical-expr/src/aggregate/bit_and_or_xor.rs

```diff
@@ -718,21 +828,84 @@ impl RowAccumulator for BitXorRowAccumulator {
         Ok(())
     }
 
-    fn update_scalar_values(
-        &mut self,
-        values: &[ScalarValue],
+    fn update_single_row(
```

Comment:
As I mentioned above, I think the code would be simpler and likely faster if there wasn't a special single value case

I seems like given that you have vectorized the code for updating the aggregator values, and the group operators already have a slice if `[usize]` (aka the selection vector) of values to aggregate, maybe all callsites can use the selection vector

Reply:
I had updated the interface.